nucypher/blockchain/eth/actors.py

```diff
@@ -315,6 +320,31 @@ def deploy_network_contracts(self,
 
         return deployment_receipts
 
+    def relinquish_ownership(self,
```

Comment:
Just idea: maybe extract "owner" functions to some base actor/agent

Reply:
Yes, I *almost* wrote some kind of ownable agent mixin the other day.  Unsure if it can just be best represented as an attribute?

Reply:
I think yes. Maybe same as upgradable attribute

Reply:
See #1314